src/main/java/edu/ucsb/cs156/happiercows/entities/UserCommons.java

```diff
@@ -29,5 +29,7 @@ public class UserCommons {
   private double totalWealth;
 
   private int numOfCows;
+
+  private long totalCowHealth;
```

Comment:
Please consider instead:

private double cowHealth;

This should NOT be an integer, and it isn't a total.